stdlib/LibGit2/test/libgit2.jl

```diff
@@ -600,6 +600,7 @@ end
 end
 
 mktempdir() do dir
+    dir = realpath(dir)
```

Comment:
If this is needed on Windows, shouldn’t it be part of the mktempdir implementation?

Reply:
I think probably yes.